app/pb_kits/playbook/pb_typeahead/index.js

```diff
@@ -26,7 +26,7 @@ export default class PbTypeahead extends PbEnhancedElement {
   search() {
     if (this.searchTerm.length < this.searchTermMinimumLength) return this.clearResults()
 
-    this.toggleResultsLoadingIndicator('visible')
+    this.toggleResultsLoadingIndicator(1)
```

Comment:
Mostly, I think we want to represent booleans as either `true` or `false` vs int.